### ISSUE TITLE
Fans turn off when fan curve is applied.

### ISSUE THREAD

**Issue by Dawabenelona** [Rem0o/FanControl.Releases#283]
Hi, first of all I would like to thank you for this amazing work. I hope my donation will help you for the future development.

Regarding my problem: Whenever I apply a fan curve it turns the fan off (0 RPM). This happens with all the different curve types (linear, graph, target, flat, mix, manual control). As soon as I deactivate the fan curve it ramps the RPM up again.

Motherboard: ASUS TUF GAMING X570-PLUS (WI-FI).
Fan: be quiet! SilentWings 3 PWM High-Speed (140mm).
My fan is directly connected via 4-Pin connector to the motherboard.

I observed this problem after updating the software. Before that it worked perfectly. I tried older versions without success.

Any idea what causes this problem?

**Comment by Dawabenelona** [comment]
[UPDATE]
I did the following things without success:

1. BIOS reset
2. Disabled Q-Fan Control in BIOS
3. Enabled PWM Fan Control in BIOS
4. Killed all tasks in task manager that were somehow related to programs controlling the fans

While playing around with BIOS settings, I noticed that I couldn't even control the fans from there. I then opened Asus Armoury Crate to check if I could control the fans from there - with success. **After setting the fan speed to standard in Armoury Crate I was able to control my fans again.**
